tools/toolchain/scripts/stage4/install_scalapack.sh

```diff
@@ -39,26 +39,29 @@ case "$with_scalapack" in
       if [ -f ${scalapack_pkg} ]; then
         echo "${scalapack_pkg} is found"
       else
-        download_pkg_from_cp2k_org "${scalapack_sha256}" "${scalapack_pkg}"
+	wget https://github.com/Reference-ScaLAPACK/scalapack/archive/refs/tags/v2.2.1.tar.gz -O "${scalapack_pkg}"
+        #download_pkg_from_cp2k_org "${scalapack_sha256}" "${scalapack_pkg}"
```

Comment:
I've uploaded [scalapack-2.2.1.tar.gz](https://www.cp2k.org/static/downloads/scalapack-2.2.1.tar.gz) to our server. So you can now use `download_pkg_from_cp2k_org` again.